mern/client/src/screens/StartingIntent/StartingIntent.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
This is a good start to the start intent screen.

There are currently place-holders for the question and intent. Remember to have the back-end call to get the manipulated/parsed transcript data to be displayed instead.

Reply:
Noted and added to Trello board as an issue